hive-bigquery-parent/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (C) 2022 Google, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud.hive</groupId>
+  <artifactId>hive-bigquery-parent</artifactId>
+  <version>${revision}</version>
+  <packaging>pom</packaging>
+  <name>Hive BigQuery Connector Build Parent</name>
+  <description>Parent project for all the Hive BigQuery Connector artifacts
+  </description>
+
+  <properties>
+    <revision>2.0.0</revision>
```

Comment:
default revision should be 2.0.0-SNAPHOT

Reply:
Ok, done